pydantic/_internal/_validators.py

```diff
@@ -105,21 +104,6 @@ def _import_string_logic(dotted_path: str) -> Any:
         raise ImportError(f'Module "{module_path}" does not define a "{class_name}" attribute') from e
 
 
-def uuid_validator(__input_value: str | bytes) -> UUID:
```

Comment:
Moved inline to where it gets used. We should probably get rid of this file eventually.